### ISSUE TITLE
Crash after upgrading from `0.22.10` to `0.22.11`

### ISSUE THREAD

**Issue by Jason3S** [TypeStrong/typedoc#1852]
Thank you for the tool. It is very helpful.

## Expected Behavior

Expect that `0.22.11` should work as well as `0.22.10`.

## Actual Behavior

```
TypeDoc exiting with unexpected error:
TypeError: Cannot read properties of undefined (reading 'qualifiedName')
    at Renderer.attemptExternalResolution (./node_modules/typedoc/dist/lib/output/renderer.js:114:19)
    at Object.attemptExternalResolution (./node_modules/typedoc-plugin-markdown/dist/resources/helpers/attemptExternalResolution.js:6:28)
    at getReferenceType (./node_modules/typedoc-plugin-markdown/dist/resources/helpers/type.js:119:44)
    at ReferenceType.<anonymous> (./node_modules/typedoc-plugin-markdown/dist/resources/helpers/type.js:10:20)
    at getType (./node_modules/typedoc-plugin-markdown/dist/resources/helpers/declaration-title.js:17:41)
    at DeclarationReflection.<anonymous> (./node_modules/typedoc-plugin-markdown/dist/resources/helpers/declaration-title.js:29:21)
    at DeclarationReflection.wrapper (./node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
    at Object.eval [as main] (eval at createFunctionContext (./node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:262:23), <anonymous>:10:248)
    at main (./node_modules/handlebars/dist/cjs/handlebars/runtime.js:208:32)
    at Object.ret (./node_modules/handlebars/dist/cjs/handlebars/runtime.js:212:12)
```

## Steps to reproduce the bug

If the crash is hard to reproduce, here are some steps:

### Setup

```sh
git clone https://github.com/streetsidesoftware/cspell.git
cd cspell
git checkout dev-typedoc
npm i
cd packages/cspell-types
```

### Failing
```sh
# test building docs
npm run test-build-docs
```

### Downgrad to `0.22.10`
```sh
# test building docs
npm i -SD typedoc@0.22.10
npm run test-build-docs
```

## Environment

-   Typedoc version: 0.22.11
-   TypeScript version: 4.5.4
-   Node.js version: 16.13.1
-   OS:

**Comment by Jason3S** [comment]
This is related to: [Cannot read properties of undefined (reading 'qualifiedName') · Issue #283 · tgreyuk/typedoc-plugin-markdown](https://github.com/tgreyuk/typedoc-plugin-markdown/issues/283)

**Comment by Jason3S** [comment]
Fixed by: [Release typedoc-plugin-markdown@3.11.12 · tgreyuk/typedoc-plugin-markdown](https://github.com/tgreyuk/typedoc-plugin-markdown/releases/tag/typedoc-plugin-markdown%403.11.12)
